assets/test/ConfiguredHeadwaysForm.test.ts

```diff
@@ -284,27 +296,21 @@ describe('With Multi-sign Headway enabled', () => {
 
       await user.type(
         container.querySelector(
-          'input[name="branches.[0].morning.range_high"]',
+          'input[name="branches.[0].morning.range_low"]',
         )!,
-        '8',
+        '10',
       );
-
       await user.type(
         container.querySelector(
-          'input[name="branches.[0].morning.range_low"]',
+          'input[name="branches.[0].morning.range_high"]',
         )!,
-        '10',
+        '8',
```

Comment:
Were these switched for clarity, or because the order matters?

Reply:
Order did matter because of the quirk that you mentioned in your prior comment about the alert banner sometimes not showing up. But I just switched it back since the form behaves better with the change to use `isValid` as the condition for showing the error banner.